docs/content/references/config/index.md

```diff
@@ -0,0 +1,27 @@
+# Configuration
+
+HedgeDoc can be configured via environment variables either directly or via an `.env` file.
+
+## The `.env` file
+
+The `.env` file should be placed in the root of the project and contains key-value pairs of
```

Comment:
Not sure if "root of the project" is confusing here?

Did everybody reading this clone our repo?